src/redux/orders/orders.sagas.js

```diff
@@ -55,17 +40,24 @@ export function* handleOrdersListLoad({ payload }) {
       payload.limit,
       payload.filter.orderStatus
     );
-    if (orders.errors) {
```

Comment:
Why you remove this from saga?

Reply:
it doesn't work

Reply:
It should work if we have errors array in response.